### ISSUE TITLE
gradient_acc_steps parameter

### ISSUE THREAD

**Issue by svjan5** [plkmo/BERT-Relation-Extraction#22]
Hi,
Thanks for sharing the implementation of "Matching the Blank" paper. I just wanted to know what is the benefit of changing `gradient_acc_steps` parameter? Does it make training/convergence faster?

Thanks and regards

**Comment by plkmo** [comment]
gradient_acc_steps = 2 will wait for 2 batch passes before backpropagating gradients. Allows you to effectively increase batch size by 2 while keeping GPU memory fixed
